packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -395,23 +398,28 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
           sender: newStep.sender,
           numberVerificationPending: false,
           includeCalendarEvent: newStep.includeCalendarEvent,
+          ...(SCANNING_WORKFLOW_STEPS && didBodyChange ? { safe: true } : {}),
```

Comment:
@CarinaWolli thoughts here? Only if the body has changed we should remark the step as unsafe.

Reply:
looks good

Reply:
oh wait, why `true` here and not `false`?